app/static/src/app/store/run/run.ts

```diff
@@ -19,7 +20,14 @@ export const defaultState: RunState = {
     numberOfReplicates: 5,
     parameterSetsCreated: 0,
     parameterSets: [],
-    parameterSetResults: {}
+    parameterSetResults: {},
+    advancedSettings: {
+        [AdvancedOptions.tol]: { val: [null, null], defaults: [1, -6], standardForm: true },
```

Comment:
can you pull these from the responseTypes please (they'll need adding there). that's where we've been putting manually copied interfaces from the odin-js bits

Reply:
I have moved both AdvancedOptions and AdvancedSettingsOdin into responseTypes, I think the rest should live in non-responseTypes territory

Reply:
Ah, it was the defaults too I was looking at here. We've had a nightmare chasing defaults around the app before, they get scattered everywhere.

Reply:
Since these are fixed defaults for all apps, this seems like the right place for them .